hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/om/TestKeyManagerImpl.java

```diff
@@ -638,8 +633,6 @@ public void testInvalidPrefixAcl() throws IOException {
         .build();
 
     // add acl with invalid prefix name
-    exception.expect(OMException.class);
-    exception.expectMessage("Invalid prefix name");
     writeClient.addAcl(ozInvalidPrefix, ozAcl1);
```

Comment:
Isn't this (and `setAcl`, `removeAcl`) expected to throw exception?  I see the test passes as is, but why?  Are these writes no-ops in the test?  If so, I think we should remove them.

Reply:
It's calling the real method, but seems only `getAcl()` is validating the prefix.

https://github.com/apache/ozone/blob/11fd5b12fd73b0d47dcf53a91346d842658c56d7/hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/PrefixManagerImpl.java#L99-L100

Reply:
Hi @adoroszlai, it turns out the exception should be thrown here.
There must be some errors on the master branch, see:
https://github.com/apache/ozone/pull/4396/files#r1135258057

Reply:
> it turns out the exception should be thrown here.
> There must be some errors on the master branch, see:

HDDS-8157 is created to track this issue.